### PR TITLE
Test only for "do" syntax, instead of "let"

### PR DESCRIPTION
**Pull request by sheerun**
Currently this test verifies not only "do" syntax, but also "let".

I'm changing "let" to "var" so only "do" is tested..

### CODE REVIEW

**Review by chicoxyzzy — APPROVED** [review]
LGTM

**Comment by zloirock** [comment]
Hmm... `var` unlike `let` is not block scoped, `do` is a block. It will work, but I'm not sure that it will be a correct example.

**Comment by chicoxyzzy** [comment]
we also can just omit any variables here. @zloirock what do you think about
```js
return do {
  if (true) {
    42;
  } else {
    'Pan Galactic Gargle Blaster';
  }
} === 42;
```
?

**Comment by ljharb** [comment]
Are there any implementations that would pass this test with `var`, but not with `let`?

Unless there is one, I think leaving `let` is best.

**Comment by zloirock** [comment]
@chicoxyzzy yep, as an option.

**Comment by ljharb** [comment]
I think part of the test is that the do expression creates a scope. A block-scoped variable is required to test that. Any of these changes will be testing fewer things.

**Comment by chicoxyzzy** [comment]
@sheerun please run `npm run build` and add changes to this PR

File: data-esnext.js [hunk 1]
```diff
@@ -51,7 +51,7 @@ exports.tests = [
   spec: 'http://wiki.ecmascript.org/doku.php?id=strawman:do_expressions',
   exec: function () {/*
     return do {
-      let x = 23;
+      var x = 23;
```
Comment (ljharb) [inline data-esnext.js:54]: This must be a block-level variable to properly test block scope - in other words, it must be `let` or `const`.

Why do you think this is an important change to make? Does there exist any implementation that supports do expressions but not let/const?

**Review by ljharb — CHANGES_REQUESTED** [review]

File: data-esnext.js (hunk 1 above)
Comment (sheerun) [reply data-esnext.js:54]: I've just noticed that it might test something it's not supposed to, and sent PR. Your call

Comment (ljharb) [reply data-esnext.js:54]: In this case I'm claiming it is supposed to be testing that.

Comment (domenic) [reply data-esnext.js:54]: do expressions aren't specified, but once they are, it's not even clear they'll support var declarations inside of them.

**Pull request closed without merging**